RedefineGuide/Controllers/SurgeryController.swift

```diff
@@ -76,23 +76,48 @@ class SurgeryController: NSObject {
             return
         }
 
-        do {
-            let modelNode = try loadModel("bone") // this causes the UI to freeze.  but I tried to put it in a background task and that didn't help. we'll have to handle this later
-            modelNode.scaleToWidth(20)
-            // make it translucent
-            modelNode.opacity = 0.5
-            // rotate the model up
-            modelNode.rotate(x: 90, y: 90, z: 0)
-
-            overlayNode = modelNode // Store the reference
-            updateOverlayModelPosition()
-            scene.rootNode.addChildNode(modelNode)
-            logger.info("Added model to scene")
-        } catch {
-            logger.error("Could not add leading model: \(error.localizedDescription)")
+        DispatchQueue.global(qos: .userInitiated).async {
+            do {
+                let modelNode = try self.loadModel("bone")
+                DispatchQueue.main.async {
+                    modelNode.scaleToWidth(15)
+                    modelNode.opacity = 0.5
+                    modelNode.rotate(x: 0, y: 0, z: 0)
+
+                    // Set up materials
+                    let material = SCNMaterial()
+                    material.lightingModel = .physicallyBased
+                    material.metalness.contents = 1.0
+                    material.roughness.contents = 0.0
+                    modelNode.geometry?.firstMaterial = material
+
+                    // Set up scene lighting
+                    let lightNode = SCNNode()
+                    lightNode.light = SCNLight()
+                    lightNode.light?.type = .omni
+                    lightNode.position = SCNVector3(x: 10, y: 10, z: 10)
+                    scene.rootNode.addChildNode(lightNode)
```

Comment:
What effect does this have?  I wonder if we should be introducing artificial light into a surgical environment that is already very well lit.

Also, what went into detecting the position?  As far as I know, the further away you go from the origin, the further you go from the light.  If we want to light the `modelNode`, maybe we should make the light's position relative to the `modelNode` and adjust the position in `updateOverlayModelPosition`